src/main/java/org/simplejavamail/utils/mail/smime/SmimeKey.java

```diff
@@ -89,7 +89,7 @@ private void extractAssociatedAddresses() {
 		try {
 			X509Certificate certificate = getCertificate();
 			if (null != certificate) {
-				Principal principal = certificate.getSubjectDN();
```

Comment:
Can you elaborate on this change please?

Reply:
According to its JavaDocs, `getSubjectDN()` is deprecated:

> Deprecated
> Use getSubjectX500Principal instead. This method returns the subject as an implementation specific Principal object, which should not be relied upon by portable code.

Reply:
Dear Benny
Any news on my PR? We would like to update our dependencies and are, therefore, hoping on a new release. Thanks very much!